vaadin-dialog-flow-parent/vaadin-dialog-flow-integration-tests/src/main/java/com/vaadin/flow/component/dialog/tests/DialogWithShortcutPage.java

```diff
@@ -0,0 +1,131 @@
+package com.vaadin.flow.component.dialog.tests;
+
+import java.util.EventObject;
+import java.util.stream.Collectors;
+
+import com.vaadin.flow.component.AttachEvent;
+import com.vaadin.flow.component.ClickEvent;
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.Key;
+import com.vaadin.flow.component.ShortcutRegistration;
+import com.vaadin.flow.component.Text;
+import com.vaadin.flow.component.dialog.Dialog;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.Input;
+import com.vaadin.flow.component.html.NativeButton;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+import com.vaadin.flow.router.Route;
+
+@Route("vaadin-dialog/shortcuts")
+public class DialogWithShortcutPage extends VerticalLayout {
+
+    public static final Key SHORTCUT_KEY = Key.KEY_X;
+    public static final String SHORTCUT = SHORTCUT_KEY.getKeys().stream()
+            .collect(Collectors.joining());
+    public static final String EVENT_LOG = "event-log";
+    public static final String UI_BUTTON = "ui-button";
+    public static final String MODELESS_SHORTCUT_ON_UI = "modeless-shortcut-on-ui";
+    public static final String MODELESS_SHORTCUT_LISTEN_ON_DIALOG = "modeless-shorcut-listen-on-dialog";
```

Comment:
```suggestion
    public static final String MODELESS_SHORTCUT_LISTEN_ON_DIALOG = "modeless-shorcurt-listen-on-dialog";
```

Reply:
Now it says "shortcur" instead of "shortcut"